samples/apps/programmability/programmability.cpp

```diff
@@ -223,10 +228,19 @@ namespace programmabilityapp
         }
         // End of Authorization Check
 
-        const auto bundle = get_body(ctx);
+        const auto [format, bundle] = get_body(ctx);
```

Comment:
what is format and bundle here?

Reply:
See definition of `get_body()` on line 52, format is an enum (AuditInputFormat) and bundle is a std::span<const uint8_t> (i.e. the actual body, which could be either JSON or COSE, as indicated by format).